crates/holochain_persistence_lmdb/src/eav/lmdb.rs

```diff
@@ -31,9 +30,13 @@ impl<A: Attribute> EavLmdbStorage<A> {
         db_path: P,
         initial_map_bytes: Option<usize>,
     ) -> EavLmdbStorage<A> {
-        EavLmdbStorage {
+        Self::wrap(&LmdbInstance::new(EAV_BUCKET, db_path, initial_map_bytes))
+    }
+
+    pub fn wrap(lmdb: &LmdbInstance) -> Self {
```

Comment:
```suggestion
    pub fn wrap(lmdb: LmdbInstance) -> Self {
```
It is a code smell to take something by reference and immediately clone it. Let your caller decide if they want to clone it or not.